docs/user-guide/advanced/Pandas_API.ipynb

```diff
@@ -353,6 +353,32 @@
     "tab.size"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4023811f-0f23-4e4b-919c-c055b865d9d2",
+   "metadata": {},
+   "source": [
+    "### Table.values\n",
+    "Return a matricial representation of the DataFrame.\n",
+    "\n",
+    "Only the values in the DataFrame will be returned, the axes labels will be removed.\n",
+    "\n",
+    "**Returns:**\n",
+    "\n",
+    "| Type              | Description                                                          |\n",
+    "| :---------------: | :------------------------------------------------------------------- |\n",
+    "| List | Rows of the table, each as a list. |"
```

Comment:
For the description I would instead write something like "The values of the table as a list of list" or "The values of the table as a matrix" or something among those lines. The current description looks a bit confusing to me personally.

Reply:
I'd liked more the statement "The values of the table as a matrix", so I will use that one.